internal/pkg/util/env/env.go

```diff
@@ -26,6 +28,9 @@ const (
 
 	// ApptainerEnvPrefix Apptainer environment variables recognized prefixes for passthru to container
 	ApptainerEnvPrefix = "APPTAINERENV_"
+
+	// DefaultLocalKeyDirName represents the default local key storage folder name
+	DefaultLocalKeyDirName = "keys"
```

Comment:
This doesn't need to be global, starting with uppercase.

Reply:
thanks for the comment! and the comment is addressed in this commit 
https://github.com/apptainer/apptainer/pull/573/commits/097c89a3ed7f842874b055ecae5e22aa530246bd